### PR TITLE
A couple of warnings fixed

### PR DESCRIPTION
**Pull request by juanky201271**
Nothing serious...
- change react native clipboard to community clipboard.
- change react native side menu to react native side menu updated.
Screen shots:

Screenshot_20220427_082212
Screenshot_20220427_084840
Screenshot_20220427_084933

### CODE REVIEW

**Comment by juanky201271** [comment]
@Gunnar-Stunnar Can you review this PR? of course is something basic... but, just in case.

**Review by Gunnar-Stunnar — APPROVED** [review]
looks good! good work

**Comment by juanky201271** [comment]
My concern here is: @zancas Do you want to merge into 'dev' in this PR all of these commits?

**Comment by zancas** [comment]
There're at least two separate concerns here.

One is that we've "re-targeted" paths to be against our own `zingolabs` organization forks of several projects.

That needs to be captured by itself in a PR (and possibly a single commit).   I will take care of that.

File: rust/android/build.sh [hunk 1]
```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
-
+# docker build --tag stunnar/rustndk:latest rustNDKDocker
```
Comment (zancas) [inline rust/android/build.sh]: Is this supposed to be commented out?

**Comment by zancas** [comment]
Odd...   I still see 7 commits on this PR even though the first two have been merged into `dev` and the third is the merge commit.   I feel confused.

**Comment by zancas** [comment]
This PR includes references to three commits that are already on `dev`.

Closing in favor of #5

**Pull request closed without merging**